core/include/drivers/regulator.h

```diff
@@ -12,6 +12,18 @@
 #include <tee_api_types.h>
 #include <stdbool.h>
 #include <stdint.h>
+#include <util.h>
+
+/*
+ * Regulator property flags: related to device tree binding properties
+ *
+ * REGUL_ALWAYS_ON
```

Comment:
No need to repeat the flag name (here it's wrong ;))
```
/* Regulator property flags: related to device tree binding properties */

/* Regulator should never be disabled. DT property: regulator-always-on */
#define REGULATOR_ALWAYS_ON	BIT(0)
```

Reply:
Ok, i'll fix. Thanks.